src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VuapAssertions.java

```diff
@@ -72,4 +72,28 @@ public static void verifyVideoAdSize(AutoplayVuap vuap, double videoAdHeight,
 
     Assert.assertEquals(adSlotHeight, adSlotHeightAfterVideoClose);
   }
+
+  public static boolean isVideoAdBiggerThanImageAdOasis(double videoHeight, double imageHeight) {
+    int percentResult = (int)Math.round(100-(100/(videoHeight/imageHeight)));
+    return percentResult == PERCENTAGE_DIFFERENCE_BETWEEN_VIDEO_AND_IMAGE_AD;
+  }
+
+  public static boolean isImageAdInCorrectSize(AutoplayVuap vuap) {
+    long time = System.currentTimeMillis();
+    long endTime = time+3000;
+    while(time < endTime) {
+      if (vuap.getAdSlotHeight() == vuap.getAdSlotHeight()){
+        return true;
+      }
+
+      try {
+        TimeUnit.MILLISECONDS.sleep(200);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)